app/src/main/java/agency/five/codebase/android/movieapp/ui/favorites/FavoritesScreen.kt

```diff
@@ -0,0 +1,92 @@
+package agency.five.codebase.android.movieapp.ui.favorites
+
+import agency.five.codebase.android.movieapp.mock.MoviesMock
+import agency.five.codebase.android.movieapp.navigation.MovieDetailsDestination
+import agency.five.codebase.android.movieapp.ui.component.MovieCard
+import agency.five.codebase.android.movieapp.ui.favorites.mapper.FavoritesMapper
+import agency.five.codebase.android.movieapp.ui.favorites.mapper.FavoritesMapperImpl
+import agency.five.codebase.android.movieapp.ui.theme.MovieAppTheme
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.grid.*
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+private val FavoritesMapper: FavoritesMapper = FavoritesMapperImpl()
+
+val favoritesViewState = FavoritesMapper.toFavoritesViewState(MoviesMock.getMoviesList())
+
+@Composable
+fun FavoritesRoute(
+    onNavigateToMovieDetails: (String) -> Unit
+) {
+    val favoritesViewState by remember { mutableStateOf(favoritesViewState) }
+
+    FavoritesScreen(
+        favoritesViewState,
+        onNavigateToMovieDetails
+    )
+}
+
+@Composable
+fun FavoritesScreen(
+    favoritesViewState: FavoritesViewState,
+    onNavigateToMovieDetails: (String) -> Unit,
+) {
+    LazyVerticalGrid(
+        columns = GridCells.Fixed(count = 3),
```

Comment:
This `count` doesn't say me anything, I still don't know whether it is rows or columns count. Please extract number to `private const val` field on top of the file and name it properly. Make sure to use UPPERCASE_SNAKE_CASE for constants.